src/mail.lisp

```diff
@@ -5,7 +5,6 @@
 
 
 (reblocks-auth/providers/email/resend:define-code-sender send-code ("Ultralisp.org <noreply@ultralisp.org>" url)
-  (:p ("To log into [Ultralisp.org](~A), follow [this link](~A)."
-       url
-       url))
```

Comment:
This syntax should work when spinneret/cl-markdown asdf system is loaded: https://github.com/ruricolist/spinneret?tab=readme-ov-file#markdown

Seems there is some problem with loading this system.